src/components/App/App.js

```diff
@@ -1,12 +1,43 @@
 import logo from '../../logo.svg';
 import './App.css';
 import { Route, Routes, Link } from 'react-router-dom';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import ErrorPage from '../ErrorPage/ErrorPage.js';
 import ResultPage from '../ResultPage/ResultPage.js';
 import Form from '../Form/Form.js';
+import { getTaxRate } from '../../ApiCalls.js';
 
 function App() {
+	const [zipcode, setZipcode] = useState(0);
+	const [salary, setSalary] = useState(0);
+	const [expenses, setExpenses] = useState(0);
+	const [combinedRate, setCombinedRate] = useState(0);
+
+	function updateZipcode(zipcode) {
```

Comment:
These functions are all super similar - could there be a way to dynamically update state with just one wrapper function? I'm thinking something similar to this: https://github.com/turingschool-examples/life-list-fe/blob/fb995e483c3aa6f74897a44575e4504e906cd881/src/Form.js#L13 could help dry up your code